lib/import_js/importer.rb

```diff
@@ -93,6 +93,39 @@ def import_one_variable(variable_name)
       write_imports(variable_name, resolved_js_module)
     end
 
+    # Helper for determining uniqueness.
+    # @param import [String]
+    # @return [String]
+    def normalize_import(import)
+      const_let_var_regex = %r{
+        \A
+        (?:const|let|var)\s+ # declaration keyword
+        (.+?)                # \1 variable assignment
+        \s*=\s*
+        require\(
+          ('|")              # \2 opening quote
+          ([^\2]+)           # \3 module path
+          \2                 # closing quote
+        \);?
+        \s*
+      }xm
+
+      import_regex = %r{
+        \A
+        import\s+
+        (.*?)              # \1 variable assignment
+        \s+from\s+
+        ('|")              # \2 opening quote
+        ([^\2]+)           # \3 module path
+        \2                 # closing quote
+        ;?\s*
+      }xm
+
+      import
+        .sub(const_let_var_regex, '\1 \3')
+        .sub(import_regex, '\1 \3')
+    end
```

Comment:
Sounds good. If you plan on moving these regexes farther away from where they are used, you might consider using named capture groups instead of numbers.